client/client-lib/src/mint/mod.rs

```diff
@@ -1,4 +1,4 @@
-mod db;
+pub mod db;
```

Comment:
..maybe this should not be pub..then I'd have to remove the intra doc-link in `gateway.rs` though